utils/epinow2/constants.py

```diff
@@ -3,17 +3,22 @@
     "horizon": 14,
     "priors": {"rt": {"mean": 1.0, "sd": 0.2}, "gp": {"alpha_sd": 0.01}},
     "parameters": {
-        "path": "data/parameters.parquet",
-        "blob_storage_container": None,
+        "generation_interval": {"path": None, "blob_storage_container": None},
+        "delay_interval": {"path": None, "blob_storage_container": None},
+        "right_truncation": {"path": None, "blob_storage_container": None},
```

Comment:
`path` and `blob_storage_container` aren't needed any more?

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/blob/c9292dff551eba326dad62988beecb6b71f8a3c7/tests/testthat/data/sample_config.json#L13

They've just been nested under new keys